cognite/powerops/client/shop_result_files.py

```diff
@@ -8,14 +8,13 @@
 from pathlib import Path
 from typing import TYPE_CHECKING, Generic, Optional, Sequence, TextIO, TypeVar, Union
 
-import matplotlib.dates as mdates
 import matplotlib.pyplot as plt
```

Comment:
Lets keep this for now, but I eventually want this as an optional dependency. Which means this would have to be a local import.